extensions/tmtv/src/getHangingProtocolModule.js

```diff
@@ -244,6 +244,9 @@ const ptCT = {
           displaySets: [
             {
               id: 'ctDisplaySet',
+              options: {
+                blendMode: 'MIP',
```

Comment:
I'm confused, didn't we say the crosshairs should just set the volume to `MIP` when they set a slab of any size other than the min MPR thickness? I don't think we should have to specify MIP on the hanging protocol level for every single viewport which may be effected by the crosshairs tool at some point?